src/fsharp/infos.fs

```diff
@@ -375,6 +375,9 @@ type ValRef with
          | TSlotSig(_,oty,_,_,_,_) :: _ -> isInterfaceTy g oty
          | [] -> false)
 
+    member vref.ImplementedSlotSignatures =
+        vref.MemberInfo.Value.ImplementedSlotSigs
```

Comment:
The "Value" might fail here (MemberInfo is an option).  When the value is not a member then "ImplementedSlotSigs" in the symbols API should I think return an empty list